code/unit_tests/materials.dm

```diff
@@ -89,11 +89,12 @@
 				if(!check_state_in_icon("other[i]", mat.icon_base))
 					LAZYADD(failed, "[mat_type] - '[mat.icon_base]' - missing connective base icon state 'other[i]'")
 
-			if(mat.icon_stripe)
-				if(!check_state_in_icon("[i]", mat.icon_stripe))
-					LAZYADD(failed, "[mat_type] - '[mat.icon_stripe]' - missing directional stripe icon state '[i]'")
-				if(!check_state_in_icon("other[i]", mat.icon_stripe))
-					LAZYADD(failed, "[mat_type] - '[mat.icon_stripe]' - missing connective stripe icon state 'other[i]'")
+			if(mat.wall_paintable & PAINT_PAINTABLE)
```

Comment:
Ah yep. Regarding the state caching comment above, you can remove the state checks entirely as this unit test will fail CI if the states don't exist.